corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -458,11 +458,17 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
             return !self.isSubmitting() && self.erroredQuestions().length === 0;
         });
 
+        self.getSubmitTranslation = function () {
+            var translations = self.translations;
+            const [key, _] = Object.entries(translations).find(([k]) => k.includes("submit_label"));
```

Comment:
I see the linter complaining here too -- is it possible to do this with `Object.keys()` rather than `Object.entries()` to avoid the unused `_`?

Reply:
opted for this approach: https://github.com/dimagi/commcare-hq/pull/33868#:~:text=of%20unused%20variable-,f3997f9,-Merge%20state